index.html

```diff
@@ -1,330 +1,355 @@
 <!DOCTYPE html>
-<html lang="en">
+<html>
 <head>
 	<meta charset="utf-8">
-	<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
```

Comment:
These `<meta>` tags are necessary for mobile view. I don't think your new website works for mobile view. Please work on that, as a new website should not break anything that used to work previously.